% $Log: abstract.tex,v $
% Revision 1.1  93/05/14  14:56:25  starflt
% Initial revision
% 
% Revision 1.1  90/05/04  10:41:01  lwvanels
% Initial revision
% 
%
%% The text of your abstract and nothing else (other than comments) goes here.
%% It will be single-spaced and the rest of the text that is supposed to go on
%% the abstract page will be generated by the abstractpage environment.  This
%% file should be \input (not \include 'd) from cover.tex.
% TODO pasted from CSHL abstract. edit this for here! 350 words, outline:
% P1: intro, focus on STRs as model for complex variation
% P2: Aim 1/2 (lobSTR + visualization + catalog)
% P3: Aim 3 (eSTRs)
% P4: Conclusion + future directions
Recent studies have made substantial progress in identifying genetic variants associated with disease and molecular phenotypes in humans. However, these studies have primarily focused on single nucleotide polymorphisms (SNPs), ignoring more complex variants that have been shown to play important functional roles. Here, I focus on short tandem repeats (STRs), one of the most polymorphic and abundant classes of genetic variation. I will first present lobSTR, a novel method for genotyping STRs from whole-genome sequencing datasets. Next, I will describe insights into population-wide trends of STR variation revealed by applying lobSTR to thousands of sequencing datasets to generate the largest and highest quality STR catalog to date. I will then show how we used this catalog to conduct a genome-wide analysis of the contribution of STRs to gene expression in humans. This survey revealed that STRs explain 10-15\% of the heritability of expression mediated by all common cis variants and potentially play an important role in clinically relevant conditions. Finally, I will discuss preliminary analyses incorporating functional genomics data with high quality complex variant genotypes to predict and validate the function of non-coding variants driving common human diseases. Altogether, these results highlight the putative phenotypic contribution of complex variants and the opportunity for a wealth of genetic discoveries to be gained by expanding analyses to less understood regions of the genome.