\iffalse \bibliography{MGymrekRefs.bib} \fi

\chapter{Abundant contribution of short tandem repeats to gene expression variation in humans}

\hzline

Most of this chapter was first published as:

\begin{itemize}
\item[] \textbf{Gymrek M}, Willems TF, Guilmatre A, Zeng H, Markus B, Georgiev S, Daly MJ, Price AL, Pritchard JK, Sharp AJ, Erlich Y. Abundant contribution of short tandem repeats to gene expression ariation in humans. \emph{Nature Genetics}. (2015).
\end{itemize}

\hzline

\textbf{Abstract:} The contribution of repetitive elements to quantitative human traits is largely unknown. Here, we report a genome-wide survey of the contribution of Short Tandem Repeats (STRs), one of the most polymorphic and abundant repeat classes, to gene expression in humans. Our survey identified 2,060 significant expression STRs (eSTRs). These eSTRs were replicable in orthogonal populations and expression assays. We used variance partitioning to disentangle the contribution of eSTRs from linked SNPs and indels and found that eSTRs contribute 10\%-15\% of the cis-heritability mediated by all common variants. Further functional genomic analyses showed that eSTRs are enriched in conserved regions, co-localize with regulatory elements, and can modulate certain histone modifications. By analyzing known GWAS hits and searching for new associations in 1,685 deeply-phenotyped whole-genomes, we found that eSTRs are enriched in various clinically-relevant conditions. These results highlight the contribution of short tandem repeats to the genetic architecture of quantitative human traits.

\section{Introduction}
In recent years, there has been tremendous progress in identifying genetic variants that affect expression of nearby genes, termed cis expression quantitative trait loci (cis-eQTLs). Multiple studies have shown that disease-associated variants often overlap cis-eQTLs in the affected tissue \cite{MoffattKabeschLiangEtAl2007,BarrettHansoulNicolaeEtAl2008,ArdlieDelucaSegreEtAl2015}. These observations suggest that understanding the genetic architecture of the transcriptome may provide insights into the cellular-level mediators underlying complex traits \cite{NicaMontgomeryDimasEtAl2010,NicolaeGamazonZhangEtAl2010,WardKellis2012}. So far, eQTL-mapping studies have mainly focused on SNPs and to a lesser extent on bi-allelic indels and CNVs as determinants of gene expression \cite{StrangerNicaForrestEtAl2007,GrundbergSmallHedmanEtAl2012,LappalainenSammethFriedlanderEtAl2013}. However, these variants do not account for all of the heritability of gene expression attributable to cis-regulatory elements as measured by twin studies, leaving on average about 20-30\% unexplained \cite{GrundbergSmallHedmanEtAl2012,WrightSullivanBrooksEtAl2014}.  It has been speculated that such heritability gaps could indicate the involvement of repetitive elements that are not well tagged by common SNPs \cite{ManolioCollinsCoxEtAl2009,PressCarlsonQueitsch2014}. 

To augment the repertoire of eQTL classes, we focused on Short Tandem Repeats (STRs), one of the most polymorphic and abundant types of repetitive elements in the human genome \cite{Ellegren2004,GemayelVincesLegendreEtAl2010}. These loci consist of periodic DNA motifs of 2-6bp spanning a median length of around 25bp. There are about 700,000 STR loci covering almost 1\% of the human genome. Their repetitive structure induces DNA-polymerase slippage events that add or delete repeat units, creating mutation rates that are orders of magnitude higher than those of most other variant types \cite{WeberWong1993,Ellegren2004}. Over 40 Mendelian disorders, such as Huntingtonâ€™s Disease, are attributed to STR mutations, most of which are caused by large expansions of trinucleotide coding repeats \cite{Mirkin2007}. 

Several properties of STRs suggest they may play a regulatory role. In vitro studies have shown that STR variations can modulate the binding of transcription factors \cite{ContenteDittmerKochEtAl2002,MartinMakepeaceHillEtAl2005}, change the distance between promoter elements \cite{WillemsPaulHeideEtAl1990,YogevRosengartenWatson-McKownEtAl1991}, alter splicing efficiency \cite{HefferonGromanYurkEtAl2004,HuiHungHeinerEtAl2005}, and induce irregular DNA structures that may modulate transcription \cite{RothenburgKoch-NolteRichEtAl2001}. In vivo experiments have reported specific examples of STR variations that control gene expression across a wide range of taxa, including Haemophilus influenza \cite{WeiserLoveMoxon1989}, Saccharomyces cerevisiae \cite{VincesLegendreCaldaraEtAl2009}, Arabidopsis thaliana \cite{SureshkumarTodescoSchneebergerEtAl2009}, and vole \cite{HammockYoung2005}. Recent studies reported that dinucleotide repeats are a hallmark of enhancers in Drosophila and are enriched in predicted enhancers in humans \cite{Yanez-CunaArnoldStampfelEtAl2014}. Human promoters also disproportionately harbor STRs \cite{SawayaBagshawBuschiazzoEtAl2013} and the presence of STRs in promoters or transcribed regions greatly increases the divergence of gene expression profiles across great apes \cite{SonayCarvalhoRobinsonEtAl2015}, suggesting that STRs play a key role in the evolution of expression. Several candidate-gene studies in human indeed reported that STR variations modulate gene expression \cite{GebhardtZankerBrandt1999,ShimajiriArimaTanimotoEtAl1999,WarpehaXuLiuEtAl1999,ContenteDittmerKochEtAl2002,RockmanWray2002} and alternative splicing \cite{HuiStanglLaneEtAl2003,HefferonGromanYurkEtAl2004,SathasivamNeuederGipsonEtAl2013}. In one example, a recent study found that the underlying mechanism behind a GWAS signal for Ewing Sarcoma is a sequence variant in an AAGG repeat that increases the binding of the EWSR1-FLI1 oncoprotein resulting in EGF2 overexpression \cite{GrunewaldBernardGilardi-HebenstreitEtAl2015}. Despite these accumulating lines of evidence, there has been no systematic evaluation of the contribution of STRs to gene expression in humans. 

To this end, we conducted a genome-wide analysis of STRs that affect expression of nearby genes, termed expression STRs (eSTRs), in lymphoblastoid cell lines (LCLs), a central ex-vivo model for eQTL studies. Next, we used a multitude of statistical genetic and functional genomics analyses to show that hundreds of these eSTRs are predicted to be functional. Finally, we tested the involvement of eSTRs in clinically relevant phenotypes.
