\iffalse \bibliography{MGymrekRefs.bib} \fi

\chapter{The landscape of human STR variation}
\label{chap:catalog}

\hzline

Much of this chapter was first published as:

\begin{itemize}
\item[] Willems TF, \textbf{Gymrek M}, Highnam G, The 1000 Genomes Project, Mittelman D, Erlich Y. The
  Landscape of Human STR Variation. \emph{Genome Res}. (August 2014).
\end{itemize}

\hzline

\textbf{Abstract:} Short tandem repeats are among the most polymorphic loci in the human genome. These loci play a role in the etiology of a range of genetic diseases and have been frequently utilized in forensics, population genetics, and genetic genealogy. Despite this plethora of applications, little is known about the variation of most STRs in the human population. Here, we report the largest-scale analysis of human STR variation to date. We collected information for nearly 700,000 STR loci across over 1,000 individuals in phase 1 of the 1000 Genomes Project. Extensive quality controls show that reliable allelic spectra can be obtained for close to 90\% of the STR loci in the genome. We utilize this call set to analyze determinants of STR variation, assess the human reference genome's representation of STR alleles, find STR loci with common loss-of-function alleles, and obtain initial estimates of the linkage disequilibrium between STRs and common SNPs. Overall, these analyses further elucidate the scale of genetic variation beyond classical point mutations.

\section{Introduction}
STRs are abundant repetitive elements that are comprised of recurring DNA motifs of 2- 6 bases. These loci are highly prone to mutations due to their susceptibility to slippage events during DNA replication \cite{Ellegren2004}. To date, STR mutations have been linked to at least 40 monogenic disorders \cite{PearsonNicholEdamuraCleary2005,Mirkin2007}, including a range of neurological conditions such as Huntington's disease, amyotrophic lateral sclerosis, and certain types of ataxia. Some disorders, such as Huntington's disease, are triggered by the expansion of a large number of repeat units. In other cases, such as oculopharyngeal muscular dystrophy, the pathogenic allele is only two repeat units away from the wild-type allele \cite{BraisBouchardXieEtAl1998}. In addition to Mendelian conditions, multiple studies have suggested that STR variations contribute to an array of complex traits \cite{GemayelVincesLegendreEtAl2010}, ranging from the period of the circadian clock in Drosophila \cite{SawyerHennessyPeixotoEtAl1997} to gene expression in yeast \cite{VincesLegendreCaldaraEtAl2009} and splicing in humans \cite{HefferonGromanYurkEtAl2004,SathasivamNeuederGipsonEtAl2013}.

Beyond their importance to medical genetics, STR variations convey high information content due to their rapid mutations and multi-allelic spectra. Population genetics studies have utilized STRs in a wide-range of methods to find signatures of selection and to elucidate mutation patterns in nearby SNPs \cite{TishkoffVarkonyiCahinhinanEtAl2001,SunHelgasonMassonEtAl2012}. In DNA forensics, STRs play a significant role as both the US and the European forensic DNA databases rely solely on these loci to create genetic fingerprints \cite{KayserKnijff2011}. Finally, the vibrant genetic genealogy community extensively uses these loci to develop impressive databases containing lineages for hundreds of thousands of individuals \cite{KhanMittelman2013}.

Despite their utility, systematic data about the landscape of STR variations in the human population is far from comprehensive. Currently, most of the genetic information concerns a few thousand loci that were part of historical STR linkage and association panels in the pre SNP-array era \cite{BromanMurraySheffieldEtAl1998,TamiyaShinyaImanishiEtAl2005} and several hundred loci involved in forensic analysis, genetic genealogy, or genetic diseases \cite{RuitbergReederButler2001,PearsonNicholEdamuraCleary2005}. In total, there are only 5,500 loci under the microsatellite category in dbSNP139. For the vast majority of STR loci, little is known about their normal allelic ranges, frequency spectra, and population differences. This knowledge gap largely stems from the absence of high-throughput genotyping Downloaded from genome.cshlp.org on December 8, 2015 - Published by Cold Spring Harbor Laboratory Press techniques for these loci \cite{JorgensonWitte2007}. Capillary electrophoresis offers the most reliable method to probe these loci, but this technology scales poorly. More recently, several studies have begun to genotype STR loci with whole-genome sequencing datasets obtained from long read platforms such as Sanger sequencing \cite{PayseurJingHaasl2011} and 454 \cite{MollaDelcherSunyaevEtAl2009}. However, due to the relatively low throughput of these platforms, these studies analyzed STR variations in only a few genomes.

Illumina sequencing has the potential to profile STR variations on a population-scale. However, STR variations present significant challenges for standard sequence analysis frameworks \cite{TreangenSalzberg2012}. In order to reduce computation time, most alignment algorithms employ heuristics that reduce their tolerance to large indels, hampering alignment of STRs with large contractions or expansions. In addition, due to the repetitive nature of STRs, the PCR steps involved in sample preparation induce in vitro slippage events \cite{HaugeLitt1993}. These events, called stutter noise, generate erroneous reads that mask the true genotypes. Because of these issues, previous largescale efforts to catalog genetic variations have omitted STRs from their analyses \cite{AbecasisAltshulerAutonEtAl2010} and early attempts to analyze STRs using the 1000 Genomes data were mainly focused on exonic regions \cite{McIverFondonSkinnerEtAl2011} or extremely short STR regions with a relatively small number of individuals based on the native indel callset \cite{AnandaWalshJacobEtAl2012}.

In our previous studies, we created publicly available programs that specialize in STR profiling using Illumina whole-genome sequencing data \cite{GymrekGolanRossetEtAl2012,HighnamFranckMartinEtAl2013}. Recently, we deployed one of these tools (lobSTR) to accurately genotype STRs on the Y chromosome of anonymous individuals in the 1000 Genomes Project to infer their surnames \cite{GymrekMcGuireGolanEtAl2013}, demonstrating the potential utility of STR analysis from Illumina sequencing. Here, we used these tools to conduct a genome-wide analysis of STR variation in the human population using the sequencing data of the Phase 1 of the 1000 Genome Project.

\section{Results}
\subsection{Identifying STR loci in the human genome}
The first task in creating a catalog of STR variation is to determine the loci in the human reference that should be considered as STRs. This problem primarily stems from the lack of consensus in the literature as to how many copies of a repeat distinguish an STR from genomic background \cite{LeclercqRivalsJarne2007,FondonMartinRichardsEtAl2012,SchaperKajavaHauserEtAl2012}. For example, is $(AC)_2$ an STR? What about $(AC)_3$ or $(AC)10$? Furthermore, as sporadic bases can interrupt repetitive DNA sequences, purity must also be taken into account when deciding whether a locus is a true STR.  

We employed a quantitative approach to identify STR loci in the reference genome. Multiple lines of study have proposed that the birth of an STR is a relatively rare event with complex biology \cite{Ellegren2004,GemayelVincesLegendreEtAl2010,AnandaWalshJacobEtAl2012,BuschiazzoGemmell2006,OliverSlashinskiWangEtAl2012,KelkarEckertChiaromonteEtAl2011}. The transition from a proto-STR to a mature STR requires non-trivial mutations such as the arrival of a transposable element, slippage-induced expansion of the proto-STR, or precise point mutations that destroy non-repetitive gaps between two short repeat stretches. Based on these observations, it was suggested that randomly-shuffled DNA sequence should rarely produce mature STR sequences and therefore can be used as negative controls for STR discovery algorithms \cite{GemayelVincesLegendreEtAl2010,SchaperKajavaHauserEtAl2012}. We utilized this approach to identify STR loci in the human genome while controlling the false positive rate \cite{SuppWillemsGymrekHighnamEtAl2014}. We first integrated the purity, composition, and length of putative STRs in the genome into a score using Tandem Repeats Finder [TRF] \cite{Benson1999}. Then, we generated random DNA sequences using a second-order Markov chain with similar properties to the human genome (i.e. nucleotide composition and transition frequencies). We tuned the TRF score threshold such that only 1\% of the identified STR loci in our collection were expected to be false positives. The resulting score thresholds were in good qualitative agreement with those previously produced using a variety of alternative experimental and analytical methods \cite{LaiSun2003,FondonMartinRichardsEtAl2012,KelkarStrubczewskiHileEtAl2010} \cite{SuppWillemsGymrekHighnamEtAl2014}. We then evaluated the false negative rate of our catalog using two methods \cite{SuppWillemsGymrekHighnamEtAl2014}. First, we collected a preliminary call set of repeat number variability across the human population with a highly permissive definition of STR loci. We found that our catalog misses only $\sim 1$\% of loci that exhibited repeat variability in the permissive call set \cite{SuppWillemsGymrekHighnamEtAl2014}. Second, we also collated a set of about 850 annotated bona-fide STR loci, mainly from the CODIS forensic panel and Marshfield linkage panel. Only 12 (1.4\%) of these markers were not included in the catalog based on the TRF score threshold. The results of the two validation methods suggest that our catalog includes $\sim$99\% of the true STRs in the genome and has a false negative rate of about 1\%. 

Overall, our STR reference includes approximately 700,000 loci in the human genome. About 75\% of these loci are di and tetra-nucleotide STRs, while the remaining loci are tri, penta and hexa-nucleotide STRs \cite{SuppWillemsGymrekHighnamEtAl2014}. Approximately 4,500 loci overlap coding regions, 80\% of which have either trimeric or hexameric repeat units.  In addition, the reference contains a roughly equal proportion of interrupted and uninterrupted microsatellites.  

\subsection{Profiling STRs in 1000 Genomes samples}
We collected variations for these 700,000 STR loci using 1,009 individuals from phase 1 of the 1000 Genomes Project (\textbf{Methods \ref{sec:catmet}, \cite{SuppWillemsGymrekHighnamEtAl2014}}). These samples span populations from five continents and were subject to low coverage ($\sim$5x) whole-genome sequencing using 76bp and 100bp Illumina paired-end reads. In addition, high coverage exome sequencing data was available for 975 of these samples and was integrated with the whole-genome raw sequencing files.  

We tested two distinct STR genotyping pipelines designed to analyze high-throughput sequencing data, namely lobSTR \cite{GymrekGolanRossetEtAl2012} and RepeatSeq \cite{HighnamFranckMartinEtAl2013}. Briefly, lobSTR utilizes the non-repetitive flanking regions surrounding STRs to align reads and assess their allele lengths, while RepeatSeq utilizes Bayesian model selection to genotype previously aligned STR-containing reads. Despite significant methodological differences, the STR genotypes from the two tools were quite concordant and matched for 133,375,900 (93\%) out of the 143,428,544 calls that were reported by both tools. We tested multiple methods to unify the two call sets in order to further improve the quality \cite{SuppWillemsGymrekHighnamEtAl2014}. However, none of these integration methods improved the accuracy. Since the lobSTR calls showed better quality for highly polymorphic STRs, we proceeded to analyze STR variations using only this call set.  

On average, we collected STR genotypes for approximately 530 individuals per locus (\textbf{Figure \ref{fig:catfig1}a}) and 350,000 STR loci per individual (\textbf{Figure \ref{fig:catfig1}b}), accumulating a total of about 350 million STR genotypes in the catalog. We examined the marginal increase in the number of covered STR loci as a function of sample size (\textbf{Methods \ref{sec:catmet}, Figure \ref{fig:catfig1}c}). Our analysis shows that after analyzing 100 samples, there is a negligible increase in the number of genotyped STRs. However, even with all of the data, 3\% of STR loci are persistently absent from the catalog. The average reference allele length of the missing STR loci was 182bp compared to 31bp for the rest of the reference, suggesting that the missing STR loci have allele lengths beyond the read length of Illumina sequencing. We also examined the marginal increase of polymorphic STR loci with minor allele frequencies (MAF) greater than 1\%. Again, we observed an asymptote after approximately 100 samples. These saturation analyses suggest that with the current sample size, the STR variation catalog virtually exhausted all loci with MAF$>$1\% that can be observed with 100bp Illumina reads and our analysis pipeline.  

\begin{figure}[h!]
\centering
\label{fig:catfig1}
\includegraphics[width=0.5\textwidth]{Figures/Chapter3/Fig1.jpg}
\caption{\textbf{Call set statistics} \textbf{(A) Distribution of the number of called samples per locus.} The average is 528 samples per STR with a standard deviation of 231 \textbf{(B) Distribution of the number of called loci per sample.} The average is 349,892 STR per sample with a standard deviation of 145,135 \textbf{(C) Saturation curves for the catalog.} The number of called loci (green) rapidly approaches the total number of STRs in the genome (red line). The number of called loci with a MAF$>$1\% (blue) saturates after 100 samples and far exceeds the number of STR variants in dbSNP (grey line close to the Xaxis).}
\end{figure}

The full catalog of STR variations is publicly available at \url{http://strcat.teamerlich.org} in VCF format. In addition, the website provides a series of graphical interfaces to search for STR loci with specific biological properties such as distance to splice sites, obtain summary statistics such as allelic spectrum and heterozygosity rates, and view the supporting raw sequencing reads. 

\subsection{Quality assessment}
To initially assess the accuracy of our STR calls, we first examined patterns of Mendelian inheritance (MI) of STR alleles for three low-coverage trios present in the sample set. In total, we accumulated half a million genotypes calls. Without any read depth threshold, 94\% of the STR loci followed MI (\textbf{Figure \ref{fig:catfig2}a}). The MI rates increased monotonically with read depth and restricting the analysis to loci with at least ten reads increased the Mendelian inheritance to over 97\%.  

Next, we compared the concordance of the calls in our catalog to those obtained using capillary electrophoresis, the gold standard for STR calling (\textbf{Methods \ref{sec:catmet}}). We focused on datasets containing Marshfield and PowerPlex Y chromosome panel genotypes that are available for a subset of the 1000 Genomes individuals. These panels ascertain some of the most polymorphic STR loci, testing our pipeline in a challenging scenario. The Marshfield capillary panel \cite{Rosenberg2006} reported 5,164 genotypes that overlapped with the lobSTR calls and pertained to 157 autosomal STRs and 140 individuals, while the PowerPlex capillary panel reported 784 genotypes that overlapped with the lobSTR calls and pertained to 17 Y-STRs and 228 individuals.  

One key question is finding an adequate cost function to assess the concordance between the STR calls. In SNPs, the proportion of mismatches is a natural measure of concordance due to their binary nature. However, for STRs, this approach assigns the same penalty for missing one repeat unit and ten repeat units. As an alternative, we focused on measuring the goodness-of-fit ($R^2$) between the STR dosages. The dosage of an STR was defined as the sum of the number of base pairs after subtracting the reference allele. For example, if the genotype was 16bp/18bp and the reference allele was 14bp, the dosage of the locus was set to 2+4=6, while for hemizygous loci the dosage was the difference from the reference allele. We focused on assessing dosage concordance because of the growing body of studies suggesting that the phenotypic impact of STRs is strongly correlated with length \cite{GebhardtZankerBrandt1999,ShimajiriArimaTanimotoEtAl1999,ContenteDittmerKochEtAl2002,HefferonGromanYurkEtAl2004}. $R^2$ confers the property that the cost is proportional to the (squared) magnitude of the error in terms of length. In addition, the $R^2$ of the dosages measures the amount of genetic variance that was recovered by lobSTR under strict additivity, which might be important for downstream association studies. 

After regressing the lobSTR dosages with the capillary dosages, the resulting goodness of fit estimators ($R^2$) were 0.71 for the autosomal genotypes and 0.94 for the Y chromosome genotypes (\textbf{Figure \ref{fig:catfig2}b; \cite{SuppWillemsGymrekHighnamEtAl2014}}). By further stratifying the autosomal calls by the capillary genotype, we found that lobSTR correctly reported 89.5\% of all homozygous loci and recovered one or more alleles for 91.5\% of all heterozygous loci, but only correctly reported both alleles for 12.8\% of all heterozygous loci \cite{SuppWillemsGymrekHighnamEtAl2014}. For the Y chromosome, 95\% of the lobSTR genotypes exactly matched the capillary genotypes for the PowerPlex Y panel \cite{SuppWillemsGymrekHighnamEtAl2014}. 

Collectively, these results suggest that the individual allele lengths are relatively accurate and that the primary source of noise is the recovery of only one STR allele for heterozygous loci, an issue known as allelic dropout. This statement is supported by the relatively good accuracy achieved for the homozygous autosomal loci and hemizygous Y chromosome loci, and the monotonically increasing relationship between heterozygote accuracy and read depth, with a heterozygote accuracy of nearly 80\% achieved for loci covered by 6 or more reads (Supplemental Figure 4). In general, allelic dropouts are quite expected given the relatively low sequencing coverage but are also known to be an issue in genotyping STRs with capillary electrophoresis \cite{PompanonBoninBellemainEtAl2005}.   

We performed various analyses that demonstrate that allelic dropouts do not hamper the ability to deduce population-scale patterns of human STR variation. First, we examined the concordance of heterozygosity rates obtained from the lobSTR and the capillary calls for Marshfield STRs in three European subpopulations (CEU, GBR and FIN). The heterozygosity rate is based on the frequency spectrum of a locus (\textbf{Methods \ref{sec:catmet}}) and should be unaffected by random allelic dropout. As expected, we found that the heterozygosity rates were highly similar between the capillary and the lobSTR results (\textbf{Figure \ref{fig:catfig2}c}). The regression slope was 0.996 and the root mean squared error (RMSE) was 0.044 based on over 200 STRs. This analysis shows that the heterozygosity estimates obtained from our call set are relatively unbiased. 

We also benchmarked the quality of population-scale patterns by comparing the allelic spectra for the Marshfield loci \cite{SuppWillemsGymrekHighnamEtAl2014}. We found that in most cases, the lobSTR and capillary spectra matched in the median and interdecile range of the reported allelic lengths. We also inspected the frequency spectra of STRs that are part of the forensic CODIS test panel using a similar procedure (\textbf{Figure \ref{fig:catfig2}d; \cite{SuppWillemsGymrekHighnamEtAl2014}}). A previous study reported the spectra of these loci by genotyping $\sim$200 Caucasians in the United States using capillary electrophoresis \cite{BudowleSheaNiezgodaEtAl2001}. Again, these comparisons resulted in similar patterns for eight of the ten analyzed markers. We found marked biases only for FGA and D18S51, with lobSTR reporting systematically shorter alleles. As the maximal allele sizes of these two loci are over 80bp, the long alleles are seldom spanned by the mixture of 76 and 100 bp Illumina reads in Phase 1, creating a bias toward shorter alleles.  

We sought to further characterize potential biases towards ascertaining shorter alleles with lobSTR and the 76bp/100bp Illumina reads. To that end, we inspected the concordance between the lobSTR calls and the NCBI reference (\textbf{Figure \ref{fig:catfig2}e}). The NCBI reference was generated by long Sanger reads and therefore should be an unbiased estimator of the most common allele in the population. In the absence of any systematic bias towards shorter alleles, the expected deviation of a lobSTR allele from the NCBI reference should be zero. On the other hand, in the presence of such a bias, the lobSTR calls should be systematically smaller than the NCBI reference and generate a negative deviation. We found that the median deviation of lobSTR was around zero for STRs with reference alleles up to 45bp. Above this threshold, we started to observe systematic deviations towards shorter alleles. The deviation did not monotonically decrease but exhibited a local maximum around 65bp, which presumably stems from the heterogeneity of the sequencing read lengths and the exhaustion of alleles that can be spanned by 76bp reads. Importantly, only 10\% of all loci in our catalog have a reference allele greater than 45bp. This implies that for the vast majority of the loci, the allelic spectra are expected to be unbiased.  

\begin{figure}[h!]
\centering
\label{fig:catfig2}
\includegraphics[width=0.5\textwidth]{Figures/Chapter3/Fig2.jpg}
\caption{\textbf{Quality assessments of the STR catalog} \textbf{(A) Consistency of lobSTR calls with Mendelian inheritance.} The blue line denotes the fraction of STR loci that followed Mendelian inheritance as a function of the read coverage threshold. The green line denotes the total number of calls in the three trios that passed the coverage threshold \textbf{(B) Concordance between lobSTR and capillary electrophoresis genotypes.} The STR calls were taken from the highly polymorphic Marshfield panel. The dosage is reported as the sum of base pair differences from the NCBI reference. The area of each bubble is proportional to the number of calls of the dosage combination and the broken line indicates the diagonal \textbf{(C) Comparison of heterozygosity rates for Marshfield panel STRs.} The color denotes the length of the median allele of the STR (dark-short; bright-long) \textbf{(D) A comparison of allelic spectra obtained by lobSTR and capillary electrophoresis for a CODIS marker in European individuals.} Red: lobSTR, black: capillary electrophoresis. nlobSTR and nCapillary indicate the number of alleles called in the respective call sets. \textbf{(E) The reliable range of lobSTR allelic spectra.} The figure presents the median deviation of the lobSTR calls from the NCBI reference as a function of the NCBI reference alleles (blue curve). Negative deviations indicate a potential preference towards ascertaining shorter alleles. STRs with reference alleles of up to $\sim$45bp show very minimal deviations (yellow region) and are expected to display unbiased frequency spectra with the current read lengths. These STR loci comprise close to 90\% of the total genotyped STRs in our catalog (red curve).}
\end{figure}

\subsection{Validation using population genetics trends}
To further assess the utility of our catalog, we tested its ability to replicate known population genetics trends. We specifically wondered about the quality of the most variable STR loci in the catalog. One hypothesis is that these loci are just extreme cases of genotyping errors; an alternative hypothesis is that these loci are truly polymorphic and can provide useful observations about the underlying populations.  We first compared the heterozygosities of the 10\% most variable autosomal loci across ten different subpopulations from Africa, East Asia, and Europe. Consistent with the Out-of-Africa bottleneck \cite{StonekingKrause2011}, we found that the genetic diversity of the African subpopulations significantly exceeded those of Europe and East Asia (sign test; $p < 10^{-50}$ for any African non-African pair) (\textbf{Figure \ref{fig:catfig3}a; \cite{SuppWillemsGymrekHighnamEtAl2014}}). Second, we focused on the 100 most heterozygous autosomal loci in our catalog and inspected the ability of STRUCTURE \cite{PritchardStephensDonnelly2000} to cluster a subset of the samples into three main ancestries in an unsupervised manner. Our results show that all of these samples clustered distinctly by geographical region (\textbf{Figure \ref{fig:catfig3}b}). These analyses demonstrate that even the most variable loci in the catalog still convey valid genetic information that can be useful for population genetic analyses. Finally, we also analyzed the genetic variability of all STRs with MAF$>$1\% on the autosome, X chromosome, and Y chromosome (\textbf{Figure \ref{fig:catfig3}c}). Autosomal STRs showed the highest variability, followed by STRs on the X and the Y chromosomes. This result is consistent with the differences in the effective population sizes of these three types of chromosomes, providing an additional sanity check.  

\begin{figure}[h!]
\centering
\label{fig:catfig3}
\includegraphics[width=0.5\textwidth]{Figures/Chapter3/Fig3.jpg}
\caption{\textbf{Evaluation of the STR catalog for population genetics} \textbf{(A) Genetic diversity of the 10\% most heterozygous autosomal loci in different populations.} Yellow: European, Red: African, Blue: East Asian. The mean heterozygosities (dot) of the African subpopulations consistently exceed those of the non-African subpopulations. The whiskers extend to ± one standard deviation. See Supplemental Table 3 for population abbreviations \textbf{(B) STRUCTURE clustering based on the 100 most polymorphic autosomal STR loci.} Each subpopulation clusters tightly by geographic origin. Color labels as in \textbf{(A). (C) Average STR heterozygosity as a function of chromosome type.} Bars denote the standard error.}
\end{figure}

In summary, the multiple lines of quality assessment suggest that our catalog can be used to infer patterns of human STR variations such as heterozygosity, allelic spectra, and population structure. The most notable shortcoming of the catalog is allelic dropouts stemming from the low sequencing coverage of the 1000 Genomes. However, the experiments above suggest that valuable summary statistics can be extracted from the call set despite this caveat. 

\subsection{Patterns of STR variation}
Despite a plethora of STR studies, there is no consensus in the literature regarding the effect of motif characteristics on STR variability. The classical study by Weber and Wong \cite{WeberWong1993} originally suggested that tetranucleotide STRs mutate more rapidly than those with dinucleotide motifs based on the analysis of de-novo mutation in trios for 50 STRs. This finding was recently supported by a much larger trio-based study of nearly 2500 STRs \cite{SunHelgasonMassonEtAl2012}. However, various other studies have suggested that dinucleotides have higher mutation rates \cite{ChakrabortyKimmelStiversEtAl1997,PembertonSandefurJakobssonEtAl2009}. These disagreements may largely stem from the fact that many of these studies considered very small panels of STRs that are subject to ascertainment biases. 

\begin{figure}[h!]
\centering
\label{fig:catfig4}
\includegraphics[width=0.5\textwidth]{Figures/Chapter3/Fig4.jpg}
\caption{\textbf{Motif length and coding capabilities as determinants of STR variability.} STR heterozygosity monotonically decreases with motif length for noncoding loci and is generally reduced in non-coding (left) versus coding regions (right). The box extends from the lower to upper quartiles of the heterozygosity distribution and the interior line indicates the median. The whiskers extend to the most extreme points within 1.5*IQR of the quartiles.}
\end{figure}

To address this open question, we analyzed the sequence determinants of STR variation in our catalog. We found that for noncoding STRs, variability monotonically decreased with motif length (\textbf{Figure \ref{fig:catfig4}}). In contrast, loci with trimeric and hexameric motifs were the most polymorphic among coding STRs. These STR loci can vary without introducing frameshift mutations and therefore may be exposed to weaker purifying selection. In addition, coding STRs demonstrated significantly reduced heterozygosity compared to noncoding STRs for periods 2-5bp (Mann-Whitney U test; $p < 0.01$, \cite{SuppWillemsGymrekHighnamEtAl2014}) while hexameric STRs showed no statistically significant difference in variability between these two classes. To ensure that the dependence between motif length and heterozygosity was not confounded by length or purity biases, we stratified STR heterozygosity for pure STRs based on major allele length and motif length. This analysis still showed an inverse correlation between motif length and STR variability after stratification based on the length of the most common allele \cite{SuppWillemsGymrekHighnamEtAl2014}. In addition, this analysis showed a monotonic increase in STR variability as a function of the major allele length. Similar trends also applied for STRs with various levels of impurities, albeit with a reduced magnitude of effect and slight deviations from monotonicity \cite{SuppWillemsGymrekHighnamEtAl2014}. This observation is concordant with previous studies \cite{Ellegren2000,LaiSun2003,WhittakerHarbordBoxallEtAl2003}.

Next, we explored the effect of nucleotide composition on STR variability, another issue for which the literature has not yet reached a consensus. Previous studies have suggested that AT repeats are the least variable motif for dinucleotide STRs \cite{PembertonSandefurJakobssonEtAl2009,BachtrogAgisImhofEtAl2000}, whereas other studies claimed that AT repeats are the most variable motif \cite{SunHelgasonMassonEtAl2012,KelkarTyekuchevaChiaromonteEtAl2008}. We repeated our analysis by stratifying the STRs based on motif sequence and major allele length \cite{SuppWillemsGymrekHighnamEtAl2014}. The resulting per-motif variability results were remarkably similar with those generated using a comparison of orthologous STRs in humans and chimps \cite{KelkarTyekuchevaChiaromonteEtAl2008}. Our analysis shows that AT repeats are in general more variable that AC repeats after controlling for length of the most major allele. Similarly, for most motif lengths, STRs with an [A]nT motif tend to be more variable with long major allele lengths. However, we could not find a clear pattern across motif lengths, which is similar to the result of a previous analysis of a few dozens Y-STRs \cite{BallantyneGoedbloedFangEtAl2010}. 

\subsection{The prototypical STR}
We also wondered about the prototypical pattern of variation of an STR locus in terms of the number of alleles and their distribution. We found that 30\% of STRs have a common polymorphism with at least two alleles with frequencies above 5\%. Dinucleotide STRs have the highest rate, with 48\% of these loci displaying a common polymorphism. Moreover, 30\% of all dinucleotide STRs have more than 3 alleles with a frequency above 5\%. On the other hand, hexanucleotide STRs have the lowest common polymorphism rate, with only 13\% of these loci displaying a common polymorphism \cite{SuppWillemsGymrekHighnamEtAl2014}.  

Next, we turned to finding the prototypal allelic spectra of STRs. For each STR, we normalized the reported alleles such that they reflected the distance in number of repeats from the locus' most common allele. Then, we generated histograms that show the allelic spectra by aggregating all the alleles of STRs with the same motif length. This coarse-grained picture was similar across repeat lengths \cite{SuppWillemsGymrekHighnamEtAl2014}.  The allelic spectrum of an STR is unimodal and relatively symmetric. There is one, highly prevalent major allele, two less common alleles one repeat above and one repeat below the most common allele, and a range of rare alleles with monotonically decreasing frequency that reach over ±5 repeats from the most common allele.  

We also wondered about the population differentiation of autosomal STRs. We analyzed the $R_{st}$ \cite{Slatkin1995} for each STR between African, Asian, and European populations for STRs with heterozygosity above 5\% \cite{SuppWillemsGymrekHighnamEtAl2014}. The average Rst was between 4.5-6\% across the motif lengths and the median was around 2-3\%. In coding regions, when compared to noncoding STRs, the average $R_{st}$ was less than half for trimeric STRs but the same for hexameric STRs. Our results regarding population differentiation using STRs are reminiscent of a classical study that found similar levels of differentiation by analyzing close to 800 STR markers \cite{RosenbergPritchardWeberEtAl2002}.

\subsection{STRs in the NCBI reference and LoF analysis}
We were interested in assessing how well the most common alleles are represented in the NCBI reference (\textbf{Figure \ref{fig:catfig5}a}). We found that for over 69,000 loci (10\% of our reference set), the most common allele across the 1000 Genomes populations was at least one repeat away from the NCBI hg19 reference allele. Furthermore, the length of the most common allele only matched the length of the orthologous chimp STR 50\% of the time, reflecting the high mutability of these loci. In addition, 15,581 loci (2.25\%) in the reference genome were 10bp or more away from the most common allele in our dataset.  

For STRs in coding regions, the most common allele for 48 loci (1.1\% of coding STRs) did not match the allele present in the NCBI reference \cite{SuppWillemsGymrekHighnamEtAl2014}. In 46 out of 48 of these cases, these differences occurred for loci with trinucleotide or hexanucleotide repeats and conserved the reading frame. Moreover, for the two loci whose most common alleles were frame-shifted, these variations are unlikely to trigger the non-sense mediated decay pathway. The deletion of one 4bp unit in \emph{DCHS2} occurs a few nucleotides before the annotated RefSeq stop codon. This variation slightly alters the location of the stop codon and affects only five amino acids in the C-terminus of the protein. The 14bp deletion in \emph{ANKLE1} occurs in the last exon of the gene and introduces about 20 new amino acids into the tail of the protein. 

We also sought to identify a confident set of STR loci with relatively common loss of function (LoF) alleles. To accomplish this goal, we considered only alleles supported by at least two reads and 30\% of the total reads per called genotype. We further required that alleles be carried by 10 or more samples. Seven common LoF alleles across five genes passed this criterion: \emph{DCHS2, FAM166B, GP6, SLC9A8}, and \emph{TMEM254} \cite{SuppWillemsGymrekHighnamEtAl2014}. Out of these 5 genes, only \emph{GP6} has known implications for a Mendelian condition: a mild platelet-type bleeding disorder \cite{DumontLasneRothschildEtAl2009,HermansWittevrongelThysEtAl2009}. However, the LoF mutation in this gene resides in the last exon and is unlikely to induce the non-sense mediated decay pathway. In conclusion, the LoF analysis indicates that common STR polymorphisms rarely disrupt the reading frame.  

\subsection{Linkage disequilibrium between STRs and SNPs}
The linkage disequilibrium (LD) structure of STRs and SNPs is largely unknown. On top of recombination events, the SNP-STR LD structure also absorbs STR back mutations that could further shift these pairs of loci towards equilibrium. However, there is minimal empirical data in the literature about the pattern of this LD structure, most of which pertains to a few hundred autosomal Marshfield markers \cite{PayseurPlaceWeber2008}. To get a chromosome-wide estimate, we inspected STR loci on the hemizygous X chromosomes in male samples. Similar to the Y chromosome data, these calls do not suffer from allelic dropouts and are already phased with SNP alleles, conferring a technically reliable dataset for a chromosome-wide analysis.  

\begin{figure}[h!]
\centering
\label{fig:catfig5}
\includegraphics[width=0.5\textwidth]{Figures/Chapter3/Fig5.jpg}
\caption{\textbf{Population-scale analyses of STR variation (A) Distribution of base-pair differences between each locus' most common allele and the NCBI reference allele (B) Patterns of linkage disequilibrium for SNPs and STRs on the X chromosome.} SNP-SNP LD (dashed lines) generally exceeds SNP-STR LD (solid lines) across a range of distances and for Africans (red), Admixed Americans (green), Europeans (yellow) and East Asians (blue).}
\end{figure}

We determined the LD in terms of the $R^2$ between SNPs and STRs as a function of the distance between these markers. Only STRs and SNPs with common polymorphisms were used for the analysis. Hexameric STRs were not included due to the small sample size of 24 sites; for the other repeat motifs, we obtained hundreds to thousands of polymorphic markers. We stratified the STR-SNP LD based on the four major continental populations (Africa, Asia, Europe, and America) and contrasted them to the patterns for classical SNP-SNP LD (\textbf{Figure \ref{fig:catfig5}b}). In all cases, the SNP-SNP LD consistently exceeded mean STR-SNP LD. In addition, the African population demonstrated markedly reduced levels of SNP-STR LD and SNP-SNP LD, consistent with its larger effective population size. In general, dinucleotide STRs showed the weakest LD with nearby SNPs, which likely stems from their higher mutation rates \cite{SuppWillemsGymrekHighnamEtAl2014}. To ensure that the reduction in STR-SNP LD did not stem from comparing $R^2$ values for multiallelic and biallelic makers, we converted the STR alleles to binary markers, where the two states corresponded to the most common allele and all alternative alleles grouped together. The resulting levels of mean SNP-STR LD using these binary genotypes were nearly identical to those obtained using the multiallelic STR genotypes, indicating that this potential issue had little effect \cite{SuppWillemsGymrekHighnamEtAl2014}. 

Overall, this analysis shows that the average SNP-STR LD is approximately half of the SNP-SNP LD for variations with the same distance on the X chromosome. Since the effective population size of the X chromosome is smaller than that of the autosome, the STR-SNP LD should be even smaller on the autosome. These results suggest that association studies with tagging SNPs might be considerably underpowered to detect loci with causal STRs, specifically dinucleotide loci. 

\section{Discussion}
In the last few years, population-scale sequencing projects have made tremendous progress in documenting genetic variation across human populations. The 1000 Genomes Project has already reported approximately 40 million SNPs, 1.4 million insertion and deletions, and over 10,000 structural variants \cite{AbecasisAltshulerAutonEtAl2010} . Similar catalogs, albeit to lesser degrees of completeness, have been produced for other types of variations, such as LINE-1 insertions \cite{EwingKazazian2011} and Alu repeat variations \cite{HormozdiariAlkanVenturaEtAl2011}. Here, we presented a population-scale analysis of STR variation, adding another layer of genetic variation to existing catalogs.  

Our analysis significantly augments the level of knowledge of STR variation. Currently, dbSNP reports data for only 5,500 STR loci. Our catalog provides data on close to 700,000 STR loci, which encompasses 97\% of the STRs with motifs of 2-6bp in the genome, and contains over 300,000 STR loci with a MAF of over 1\%. One caveat of our catalog is the low reliability of individual genotypes due to allelic dropout. Nonetheless, we showed using multiple lines of analysis that summary statistic results such as frequency spectra and variation trends can be extracted from the catalog for most of the STRs. Another caveat of our catalog is that with the mixture of 76bp and 100bp sequencing reads, we could only unbiasedly ascertain the allelic spectra of about 90\% of the STRs, those with NCBI alleles of up to 45bp. To indicate this caveat, our website alerts users about a potential bias in the allelic spectrum when inspecting STRs with reference allele length beyond this range. However, we expect caveat will be alleviated in the near future with the public release of the Phase 3 data that re-sequenced a large number of Phase 1 samples with 100bp Illumina reads. We expect that this dataset will enable the generation of unbiased allelic spectra for longer STRs. 

Despite these limitations, our data provides several biological insights about STR variation. Shorter repeat motif, longer major allele, higher purity of the repeat motif, and residing outside of a coding region are all associated with an increase in STR variability. Most of the STR loci display a unimodal distribution with one very common allele and series of minor alleles with rapidly declining frequencies. This picture suggests that the stepwise mutation model largely describes the creation of new alleles in most of these loci. An open question is the exact mutation rate per generation for each locus in the genome. This question is theoretically addressable with sufficiently large number of samples by analyzing the distribution of squared differences in the repeat size between two alleles of the same locus \cite{SunHelgasonMassonEtAl2012}. However, this question cannot be addressed by our call set due to the large number of allelic dropouts that might confound such an analysis and should be addressed with datasets obtained from deeply covered genomes. 

The landscape of STR variations in the apparently healthy 1000 Genomes individuals suggests several rules of thumbs for analyzing STR variations for medical sequencing. Previous work found that membrane proteins of several pathogens contain STR loci with non-triplet motifs whose variations can be beneficial to the organism \cite{GemayelVincesLegendreEtAl2010}. These STRs confer high evolvability and adaptability of these proteins by dynamically changing the reading frame. In contrast, our data suggests that for the vast majority of human proteins, frame-shift mutations in their STR regions are not favorable. Only a handful of STRs harbor common frame-shift polymorphisms and half of the LoF alleles create a very small change in the C-terminus tail of the protein. Based on these observations, we hypothesize that most of the non-triplet coding STRs are not well tolerated and are exposed to negative selection similar to regular indels in the same region. Therefore, it is advisable for medical sequencing projects to also analyze these loci and treat them as regular LoF alleles rather than filtering them. This rule of thumb is well-echoed in a recent study of medullary cystic kidney disease type 1 that implicated the genetic pathology in a frame-shift mutation caused by a length change of a homopolymer run \cite(Kirby et al. 2013). For in-frame STR variations, our call set contains deep allelic spectra of most of these loci, providing reference distributions of apparently healthy alleles. These spectra can be used to identify atypical STR alleles and might serve as an indicator for pathogenicity.  

Although STR alleles within our call set rarely induced frame-shifts, they may introduce premature stop codons by modulating the splicing machinery. Several prior studies have observed a direct dependence of splicing efficiency on STR repeat number for \emph{CFTR} \cite{HefferonGromanYurkEtAl2004}, \emph{HTT} \cite{SathasivamNeuederGipsonEtAl2013} and \emph{NOS3} \cite{HuiStanglLaneEtAl2003}. To facilitate the analysis of such cases, we created a dedicated table on the catalog website that specifies all of the 2,237 STRs that reside within 20 base pairs of an exon-intron boundary.  

Another issue raised by our findings is the potential contribution of STRs to complex traits. Using the prototypical allelic spectra, we estimate that the average variance of STR repeat dosage is 3, 0.7, 0.4, 0.25 and 0.1 for 2-6mer STRs, respectively. Interestingly, the theoretical maximum variance for a bi-allelic SNP dosage is 0.5, six times smaller than the observed variance of dinucleotide STRs. From a theoretical statistical genetics perspective, this suggests that causal dinucleotide STR loci could explain a considerable fraction of phenotypic variance even with a relatively modest effect size. Therefore, if each STR allele in a locus slightly changes a quantitative trait in a gradual manner, the net effect on the phenotypic variance could be quite large due to the wide range of these alleles and their relatively high frequencies. Interestingly, we found that loci with dinucleotide motifs show relatively weak LD with SNPs, suggesting that GWAS studies with SNP arrays are prone to miss causal STR loci. Given the theoretical potential of STRs to contribute to phenotypic variance on one hand and their weaker LD to tagging SNPs on the other hand, one intriguing possibility is that STRs contribute to the missing heritability phenomenon of complex traits \cite{ManolioCollinsCoxEtAl2009,PressCarlsonQueitsch2014}. Our hope is that this catalog can be a reference point to test this hypothesis in future studies.  

\section{Methods}
\label{sec:catmet}

\subsection{Call set generation}
The raw sequencing files for Phase 1 of the 1000 Genomes Project were analyzed.

The lobSTR calls were generated using computing resources hosted by Amazon Web Services, GitHub version 8a6aeb9 of the lobSTR genotyper and Github version a85bb7f of the lobSTR allelotyper (\url{https://github.com/mgymrek/lobstr-code}). In particular, the lobSTR genotyper was run using the options fft-window-size=16, fft-window-step=4 and bwaq=15 and a default minimum flanking region of 8bp on both sizes of the STR region. Reads that were aligned to multiple locations were excluded from the analysis. PCR duplicates were removed from the resulting BAM files for each experiment using SAMtools \cite{LiHandsakerWysokerEtAl2009}. The individual BAMs were merged by population and the lobSTR allelotyper was run using all population BAMs concurrently, the include-flank option and version 2.0.3 of lobSTR's Illumina PCR stutter model.  

RepeatSeq (available \url{http://github.com/adaptivegenome/repeatseq}) was run using default parameters on the read alignments produced by the 1000 Genomes project. 

For both programs, we used the set of 700,000 STRs that was constructed using the second-order Markov framework \cite{SuppWillemsGymrekHighnamEtAl2014}. 

\subsection{Estimating the number of samples per locus and number of loci per sample}
The distributions of the call set parameters were smoothed using the gaussian\_kde function in the scipy.stats python package. Covariance factors of 0.01 and 0.025 were used to smooth the samples per locus and loci per sample distributions, respectively. 

\subsection{Saturation analysis}
We determined the number of loci with calls for sample subsets containing 1, 5, 10, 25, 50, 100, 250, 500, 750 and 1000 individuals. In particular, we began by randomly selecting 1 individual. To create a subset of 5 individuals, we then added 4 more random individuals and so on. For each of these sample subsets, we determined the number of loci with one or more STR calls across all samples in the subset. We repeated this whole process 10 times and used the median number of called loci across each of the 10 repetitions to create the saturation profile for all loci. 

We also determined whether loci had a MAF $>$ 1\% using all 1009 samples. We then used a procedure analogous to the one described above to select subsets of samples and determine whether or not each of these loci had a corresponding call in each subset. This procedure resulted in the saturation profile for loci with MAF $>$ 1\%. 

\subsection{Mendelian inheritance}
The three low-coverage trios contained within the dataset consisted of the following sample sets: HG00656, HG00657, HG00702 (trio 1), NA19661, NA19660, NA19685 (trio 2) and NA19679, NA19678, NA19675 (trio 3). To assess the consistency with Mendelian inheritance for a given trio, only loci for which all three samples had calls were analyzed. The coverage assigned to each trio of calls corresponded to the minimum coverage across the three samples. 

\subsection{Capillary electrophoresis comparison}
Capillary electrophoresis comparison Marshfield genotypes \cite{Rosenberg2006} were downloaded from \url{http://www.stanford.edu/group/rosenberglab/data/rosenbergEtAl2005/combined-1048.stru}. Prior to comparing genotypes, offsets were calculated to match the lobSTR calls to the length of the Marshfield PCR products. For each locus, all observed offsets were considered and scored and the optimally scoring offset across all samples was selected. In particular, for each sample, an offset was scored as a 1, 0.5, 0.25 or 0 if the lobSTR calls matched exactly, were homozygous and recovered one Marshfield allele, were heterozygous and recovered one Marshfield allele or did not match at all, respectively.  Only loci with at least 20 calls were considered in the comparison. Finally, the Pearson correlation coefficient was calculated using the sum of the allele length differences from hg19 for each locus in each sample.

Y-chromosome PowerPlex genotypes were downloaded from the 1000 Genomes Y chromosome working group FTP site. Offsets were once again calculated to match the length of the PCR products to the lobSTR calls. For each locus, the offset was calculated as the most common difference between the lobSTR and PowerPlex genotypes across samples. Only loci with at least 5 calls were considered in the comparison and the R2 was calculated between the allele length differences from hg19 for each locus in each sample. In addition, the 15 heterozygous lobSTR calls were ignored.  

Slopes and $R^2$ values for STR dosage comparisons were calculated using the linregress function in the scipy.stats package.  To mitigate the effects of outliers, we explored using regular linear regression, regression with a zero intercept and L1 penalized regression. The resulting slopes were essentially invariant to the calculation method and so statistics were reported based on traditional linear regression.  

\subsection{Heterozygosity calculations}
For each analysis, heterozygosity was calculated using the aggregated frequency spectra according to the formula $H_E = 1-\sum_i f_i^2$ where $f_i$ denotes the frequency of the $i$th allele at the locus.

\subsection{Summary statistic comparisons}
The allelic spectra of the Marshfield panel were downloaded from \url{http://research.marshfieldclinic.org/genetics/genotypingData_Statistics/markers/} and parsed using a custom Perl script (data and script available on \url{https://github.com/erlichya/str_catalog_supplemental_scripts}). Samples from the CEU, GBR, TSI, and FIN subpopulations were analyzed, and only markers with more than 50 calls were included.

We utilized all of the lobSTR calls for the CEU, GBR and FIN subpopulations to generate the lobSTR frequency spectra for each CODIS marker. Spectra were not available for 3 of the CODIS markers (D21S11, VWa, TPOX). D21S11 is too long to be spanned by Illumina reads; we had annotation difficulties for VWa and TPOX (assigning the correct STR in hg19 to the NIST STR). We then compared the available frequency spectra to those published for a Caucasian population in the United States \cite{BudowleSheaNiezgodaEtAl2001} . Because of some annotation differences between the capillary data and our reference locations, we shifted the lobSTR spectra for the D8S1179 marker by +2 repeat units. Finally, repeat lengths for which the maximum frequency was less than 2\% were not displayed. 

\subsection{Comparison of population heterozygosity}
To obtain accurate measures of heterozygosity, autosomal STR loci with less than 30 calls in any of the 10 subpopulations considered were ignored. Of the remaining loci, the 10\% most heterozygous (24,637 loci) were selected and their means and standard deviations were calculated. To determine whether a pair of populations had systematically different heterozygosity at these loci, we paired the heterozygosities for each locus and counted the number of pairs in which population A had a larger heterozygosity than population B. Ignoring the relatively small number of loci in which heterozygosities were identical, the p-value for this over/underrepresentation was then calculated using the cdf function in the scipy.stats.binom python package.  

\subsection{Deviation of lobSTR calls from the NCBI reference}
For each locus with one or more genotyped samples, we calculated the mean deviation of all samples' genotypes from the NCBI reference allele. We then pooled these per-locus deviations by reference allele length using 5bp intervals. The median within each length bin resulted in the corresponding plot of deviation vs. reference allele length. 

\subsection{Sample clustering}
STRUCTURE version 2.3.4 was utilized to perform the MCMC-based clustering. The program was run using MAXPOPS=3, BURNIN=500000, NUMREPS=1000000, no prior population information, unphased genotypes, the admixture model and no linkage disequilibrium. All 321 samples from the JPT, CHB, YRI and CEU subpopulations present in the data were clustered based on the 100 most heterozygous autosomal STRs with at least 750 called samples. Samples for which at least 75\% of the selected makers were missing calls were not including in the resulting visualization. The final triangle plot therefore contained data for 71, 80, 81, and 82 samples from the CEU, CHB, JPT and YRI populations, respectively.  

\subsection{STR variability trends}
Analysis was restricted to STRs with at least 100 called samples. STRs that overlapped an annotated RefSeq translated region were regarded as coding and these annotations were downloaded from the UCSC table browser on 2/11/2014.  The mannwhitneyu function in the scipy.stats python package was used to test for significant differences between coding and non-coding STR heterozygosity. For analyses related to allele length or purity, STRs were further restricted to those whose most common allele matched the hg19 reference to enable calculation of the locus' purity. In particular, the purity of each of these STRs was calculated as the fraction of possible positions within the STR region where the subsequent bases corresponded to a cyclic permutation of the STR's motif. The pearsonr function in the scipy.stats python package was used to calculate the Pearson correlation coefficients and their associated p-values, where each STR's length and heterozygosity represented an individual point. Finally, to generate the plots of heterozygosity vs. length, the heterozygosity for each length was calculated as the mean variability of loci within 2bp.   

\subsection{Extraction of orthologous chimp STR lengths}
Tandem Repeats Finder was run on the panTro4 assembly of the chimp genome using the default parameters and a minimum score threshold of 5. To resolve overlapping repeats, we discarded repeats with period greater than six and scanned from low to high coordinates and selected the highest scoring repeat for each overlap conflict. The chimp coordinates were mapped to hg19 coordinates using liftOver and a minimum mapping fraction of 50\%. We then intersected these coordinates with those of our reference panel and retained those loci within our panel that had a single intersecting chimp repeat whose motif matched. This resulted in orthologous chimp repeats for $\sim$83\% of our reference set of STRs.  

\subsection{$R_{ST}$ levels}
The Rst was calculated according to Slatkin \cite{Slatkin1995} using a custom Python script (code available on \url{https://github.com/erlichya/str_catalog_supplemental_scripts}). The African, European and Asian populations were comprised of the same subpopulations used throughout this study, except that the ASW population was omitted due to potential admixture. Only loci with heterozygosity above 5\% and at least 100 genotyped samples were considered. 

\subsection{Assessing linkage disequilibrium}
In order to avoid phasing SNPs and STRs, we only analyzed X chromosome genotypes in male samples. SNP calls for the corresponding samples were obtained from the 1000 Genomes Phase 1 11/23/2010 release and any pseudoautosomal loci were ignored. Analysis of STR-SNP LD was restricted to STR loci with both a heterozygosity of at least 9.5\% and at least 20 genotypes for each super population (African, East Asian, European and Ad Mixed American). For each STR that met this requirement, we identified all SNPs within 200 KB of the STR start coordinate. After filtering out SNPs with a MAF below 5\% in any of the four super populations, we calculated the level of LD for the remaining STR-SNP pairs. In particular, the $R^2$ was calculated between the SNP genotype indicator variable and the base pair difference of the STR from the reference. We also recalculated the STR-SNP LD after converting the STR alleles to binary variables, where the most common allele and all alternative alleles were mapped to 0 and 1, respectively. This binary mapping was applied to each super population individually. 

For SNP-SNP LD calculations, a seed SNP was identified for each STR meeting the aforementioned requirements. In particular, the SNP closest to the STR’s start coordinate with MAF $>$ 5\% for each super population was selected. If no such SNP existed within 1Kb, no SNP was selected and the STR was omitted from the STR-SNP LD analysis. Otherwise, we identified all SNPs within 200 KB of the seed SNP and once again removed SNPs with a MAF $<$ 5\% in any of the super populations. The LD between the seed SNP and each of these remaining SNPs was then assessed as the R2 between the two SNP genotype indicator variables.  

\section{Acknowledgements}
M.G. is supported by the National Defense Science and Engineering Graduate Fellowship. Y.E. is an Andria and Paul Heafy Family Fellow and holds a Career Award at the Scientific Interface from the Burroughs Wellcome Fund. This study was funded by a gift from Cathy and Jim Stone and an AWS Education Grant award. The authors thank Chris Taylor Smith, Wei Wei, Qasim Ayub, and Yali Xue for providing the results of the Y-STR panel for the 1000 Genomes individuals and the 1000 Genomes Project members for useful discussions. Y.E. dedicates this manuscript to Lia Erlich that was born during the last revision of this work.