\iffalse \bibliography{MGymrekRefs.bib} \fi

\chapter{Introduction}

\section{Overview}

% Goal of genomics
A central goal in genomics is to understand the genetic variants that underlie phenotypic changes and lead to disease. Recent studies have identified thousands of genetic loci associated with human phenotypes. Since the advent of next generation sequencing, the majority of genomic studies have focused on single nucleotide polymorphisms (SNPs). These are both the simplest type of variation to genotype and one of the easiest to model. However, a wide range of other classes of variants is important for controlling phenotypes.

% STRs as a model
Short tandem repeats (STRs) consist of period DNA motifs of 1-6bp and comprise of more than 1\% of the human genome. Their repetitive structure induces DNA polymerase slippage events that add or delete repeat units, resulting in mutation rates that are orders of magnitude higher than those for most other variant types. STRs are implicated in more than 40 human diseases, mostly consisting of Mendelian disorders caused by large expansions of trinucleotide repeats. Additionally, several dozen single gene studies have shown that STRs can be involved in quantitative traits including gene expression. Because of their abundance, high polymorphism rates, and previous implication of functional roles, we focused on STRs as a model to investigate the role of complex variants in human phenotypes.

% What I present as our contribution
Here I present our contributions to enable the first large scale studies of STR variation and reveal that these loci play a significant role in complex traits in humans. In the first part of this thesis, we develop novel tools for high-throughput STR analysis from next generation sequencing data (\textbf{\autoref{chap:lobstr}, \autoref{chap:pbv}}). We then apply these methods to large sequencing cohorts consisting of thousands of samples with diverse origins to provide the first population-wide catalog of hundreds of thousands of previously uncharacterized STR loci (\textbf{\autoref{chap:catalog}, \autoref{chap:sgdp}}). An important aspect of this work has been a commitment to maximize utility of our results for the wider genomics community through providing open-source software packages and online visualization tools that are alredy being utilized by other researchers. In the second part of this thesis, we interrogate the role of STRs in complex traits, focusing on gene expression as an initial phenotype (\textbf{\autoref{chap:estr}}). This study reveals more than 2,000 STRs whose lengths are correlated with gene expression (termed ``expression STRs'', or eSTRs) and shows that STRs make a significant contribution to regulating expression of nearby genes. These loci are enriched in putative regulatory regions and are predicted to modulate regulatory activity. These results highlight the contribution of STRs to the genetic architecture of gene expression and complex traits in humans.

% To frame this work
To frame this work, I first review properties and applications of STRs and what we know about their contribution to disease and molecular phenotypes in humans. Next, I describe challenges in developing high throughput methods for STR analysis and state of the art experimental and bioinformatic methods for doing so. I summarize what we have learned to date about patterns of STR variation in humans using these methods. Then, I review what has been revealed about the genetic architecture of gene expression through SNP studies and the contribution of gene regulation to human conditons. I examine evidence that suggests variants such as STRs that are not well tagged by common SNPs may play an important role in these traits. Finally, I summarize the contributions of this thesis toward enabling large scale STR analysis and highlighting an important role fo STRs in human phenotypes.

\section{STRs are abundant in the human genome}
intro: define STRs, genome composition
\subsection{Genome-wide composition of STRs}
\subsection{Applications of STRs}
forensics, genealogy, linkage analysis
\subsection{Processes leading to STR variation}

\section{Evidence from model organisms of a regulatory role for STRs}
% TODO - specific examples
% TODO - comparative

\section{STRs in human disease and phenotypic variation}
Dozens of STRs have been implicated in both disease and molecular phenotypes in humans. In nearly all of these cases, the resulting phenotype showed a quantitative relationship with the number of repeats, strongly suggesting STRs may play an important role in complex traits.

\subsection{Dozens of disorders are caused by STR expansions}
% intro, examples
STR expansions are known to cause dozens of human single-gene, Mendelian disorders \cite{Mirkin2007}, affecting more than hundreds of thousands of patients in the U.S. \cite{CoffeeKeithAlbizuaEtAl2009}. The majority of these are due to expansions of trinucleotide repeats, and nearly all affect neurological function, most with a late onset disease course. For instance, an exonic CAG repeat expansion encoding polyglutamine results in Huntington's Disease, a devastating neurological disorder \cite{Mirkin2007}; CGG expansion disrupts a methylation site on the X chrmosome leading to Fragile X Syndrome, one of the leading cuases of mental retardation in males \cite{LyonLaverYuEtAl2010}; a CUG expansion in the 3' untranslated (UTR) region \emph{DPMK} results in myotonic dystrophy \cite{BrookMcCurrachHarleyEtAl1992}, a severe multisystemic form of muscular dystrophy. 
Other classes of repeats have been implicated in STR expansion diseases. Recently, high throughput sequencing scans for causative SNPs fortuitously revealed that a hexanucleotide expansion in \emph{C9orf72} is responsible for 9p21-linked amyotrophic lateral sclerosis-frontotemporal dementia (ALS-FTD) \cite{RentonMajounieWaiteEtAl}. A summary of STR expansion diseases is given by Mirkin \cite{Mirkin2007}. %shown in \textbf{Table \ref{tab:intro1}}.

% TODO complete table
%\begin{table}[h!]
%\label{tab:intro1}
%\centering
%\begin{tabular}{c c c c c c}
%\hline
%Disorder & Gene & Repeat & Normal range & Pathogenic range & References \\
%\hline
%\hline
%\end{tabular}
%\caption{\textbf{An overview of STR expansion disorders.}}
%\end{table}

% mechanism?
The mechanisms by which most of these expansions lead to disease are still poorly understood. In cases of exonic repeat expansions, particularly polyglutamine expansions such as Huntington's Disease, it is thought that the expanded amino acid tracts form toxic aggregates that accumulate over time, consistent with the late-onset nature of these diseases \cite{MichalikVanBroeckhoven2003}. Alternatively, it was recently shown that a key factor in Huntington's Disease may be repeat-length dependent aberrant splicing of \emph{HTT} \cite{SathasivamNeuederGipsonEtAl2013}. Other proposed pathogenic mechanisms include loss of protein expression, over-expression of the wildtype protein copy, and toxic gain of function of RNAs encoding expanded repeats \cite{Pearson2011}. Recent studies have found evidence that an alternative mechanism, ``Repeat-associated non-ATG translation'', termed ``RAN-translation'', may be responsible for rendering CAG repeats toxic. Under this phenomenon, expanded CAG repeats in RNA may be translated in the absence of an ATG start codon, and may produce transcripts under all seven possible reading frames \cite{Pearson2011}. These ``RAN'' transcripts have already been identified in patients with a variety of repeat disorders, including spinocerebellar ataxia type 8 (SCA8), myotonic dystrophy type 1 (DM1), Fragile-X tremor ataxia syndrome (FXTAS), and ALS-FTD \cite{ClearyRanum2014}.

% anticipation, correlation with repeat number
One hallmark of repeat disorders is the phenomenon of ``anticipation'' in which the severity of the condition tends to increase and the age of onset tends to decrease with each generation. This generally corresponds to an in increase in repeat number for the expanded allele in each generation. Many of these repeats, experience a bi-phasic mutation process: under a certain number of repeats the region is stable with relatively low mutation rate. However, once a threshold length has been passed, the repeat will have an extremely high mutation rate and tend toward massive expansions \cite{BourgeoisCoffeyRiveraEtAl2009}. In many cases, the number of repeats is directly related to phenotype. For instance, there is a negative linear relationship between CAG repeat number and age of onset of Huntington's Disease \cite{Rubinsztein2002}. Interestingly, this suggests that unlike point mutations, which can serve as an ``on/off'' switch for Mendelian and other disorders, repeats possess a more fine-tunable mechanism to affect phenotype by adjusting the number of repeats on a quantitative, rather than a binary, scale.

% polymorphism in population 
Many such pathogenic STR expansions have been characterized, clearly indicating a biological function for at least a subset of repetitive elements. However, the majority of repeats remain uncharacterized and little is known about the extent of polymorphism and allele ranges at these and other STRs in healthy individuals. As mentioned above, STRs are prone to replication slippage events that cause them to mutate rapdily. Whereas intermediate length STRs of $\sim<$200bp tend to mutate in a stepwise fashion. longer repeats such as those involved in expansion disorders may for unusual non-B DNA structures \cite{KozlowskiSobczakKrzyzosiak2010} that cause rapid expansions, leading to unusual behavior in pathogenic conditions. We hypothesize that whereas longer unstable repeats radically disrupt function of a given locus, leading to severe mono-genic disorders, STRs in the intermediate range length may be responsible for fine-tuning genomic regulation and for contributing to more incremental variation, which could make an important contribution to more complex traits in humans.

\subsection{STRs in complex human traits}
Little is known about the role of STRs in more complex, polygenic traits. This is largely due to the fact that until recently there was limited ability to systematically profile these loci on a large scale (see \ref{sec:intromethods}) and because SNP-based studies have limited ability to capture STR associations (see \ref{sec:introarch}). However, a small number of STR associations with complex traits have been reported: repeat length in the first exon of the androgen receptor correlates with risk of hepatocellular carcinoma risk in women \cite{YuYangYangEtAl2002}; a TC repeat in \emph{HMGA2} is associated with uterine leiomyomata and decreased height \cite{HodgeTCuencoHuyckEtAl2009}; a CAG repeat in \emph{KCNN3} is associated with cognitive performance in schizophrenia \cite{GrubeGerchenAdamcioEtAl2011}. However, no study has systematically evaluated the role of STRs in complex traits.

\subsection{Mechanisms for STR involvement in genome regulation}
STRs are found in at least 5\% of human protein-coding genes \cite{ODushlaineEdwardsParkEtAl2005} and are abundant in intragenic regions and UTRs \cite{LiKorolFahimaEtAl2004}. These rapidly evolving elements provide an evolutionary substrate to incrementally affect gene activity without introducing major sequences changes. STRs have been demonstrated or hypothesized to affect gene regulation in several ways, summarized below. % and in \textbf{Table \ref{tab:intro2}}.

\subsubsection{Transcribed STRs}
% exonic
As described above, many putative pathogenic STRs lie in coding regions and may lead to disease through mechanisms such as protein or RNA aggregation, RAN-translation, or aberrant splicing. Alternative mechanisms could allow exonic STRs to affect protin function. For instance, STR mutations in coding regions could function to silence genes by introducing frameshift mutations leading to premature stop codons \cite{GemayelVincesLegendreEtAl2010}, especially in the case of non tri- or hexa-nucleotide repeats. Additionally, expansions or contractions could alter spacing between protein domains, leading to a change in function.

% intronic
Intronic repeats may also affect gene regulation. Lengths of intronic STRs have been shown to affect gene expression \cite{GebhardtZankerBrandt1999}, in some cases through altering transcription factor binding sites. For example, a TATC repeat in \emph{TH} affects binding of the transcription factor ZNF191 \cite{AlbaneseBiguetKieferEtAl2001}. Several studies have also shown that intronic STRs may regulate splicing efficiency in a repeat-specific manner. For instance, the length of an intronic TG repeat in \emph{CFTR} is directly related with inclusion of the adjacent exon \cite{HefferonGromanYurkEtAl2004}, which is likely to be due to effects of the repeat on RNA secondary structure. An intronic CA repeat in \emph{eNOS} was shown to regulate splicing by affecting binding of the splicing factor HnRNP L \cite{HuiStanglLaneEtAl2003}, and it was later shown that intronic CA repeats may have a widespread effect on splicing \cite{HuiHungHeinerEtAl2005}.

% UTRs
Finally, STRs in UTRs may affect gene regulation. For example, a CTG/CAG repeat in the 3'UTR of \emph{DMPK} is implicated in the repeat expansion disorder myotonic dystrophy 1, and is thought to sequester splicing factors leading to aberrant splicing \cite{KoscianskaWitkosKozlowskaEtAl2015}. 3'UTR repeats may also harbor microRNA binding sites, affecting gene regulation.

\subsubsection{STR lengths influence promoter and enhancer activity}
In addition to transcribed regions, STRs can affect the function of genomic regulatory elements through several mechanisms. AC dinucleotides are over-represented in predicted \emph{cis}-regulatory elements \cite{RockmanWray2002}. Recently, it was shown that dinucleotide repeats are a hallmark of enhancer elements in \emph{Drosophila} and human cell lines \cite{Yanez-CunaArnoldStampfelEtAl2014}. This suggests they may provide an abundant source of transcriptional regulation in these elements. Heidari, \emph{et al.} \cite{HeidariNarimanSalehFamEsmaeilzadeh-GharehdaghiEtAl2012} demonstrated that a GA repeat in the promoter of \emph{SOX5} can affect nucleosome processing, affecting downstream gene expression. This finding is supported by a study in yeast that found variation in repeat length had a strong effect on nucleosome positioning and gene expression in 25 of 33 randomly chosen STR-containing promoters \cite{VincesLegendreCaldaraEtAl2009}. STRs may also bind transcription factors and create a number of binding sites dependent on the number of repeats. Guillon \emph{et al.} found that the oncogenic EWSR1-FLI1 fusion protein formed in Ewing Sarcoma preferentially binds GGAA repeats \cite{GuillonTirodeBoevaEtAl2009}. Interestingly, a recently study found that a reported genome-wide association study (GWAS) signal for Ewing Sarcoma actually points to a SNP for which the alternate allele joins two adjacent GGAA repeats into one long repeat tract, resulting in overexpression of the nearby gene \emph{EGR2} \cite{GrunewaldBernardGilardi-HebenstreitEtAl2015}. In another example, a pentanucleotide repeat in the promoter of \emph{PIG3} creates a varying number of \emph{p53} binding sites, affecting downstream expression \cite{ContenteDittmerKochEtAl2002}.

% TODO complete table
%\begin{table}[h!]
%\centering
%\label{tab:intro2}
%\begin{tabular}{c c c c c}
%\hline
%Gene & STR location & Tissue & Direction of effect & Reference \\
%\hline
%\hline
%\end{tabular}
%\caption{\textbf{STRs implicated in gene regulation. Full references given in the main text}}
%\end{table}

Together, these examples show clear evidence that STRs may regulate quantitative traits, such as gene expression, transcription factor binding, and splicing efficiency, in a repeat-dependent manner and therefore are prime candidates for contributing to complex traits in humans.

\section{Methods for genotyping STRs}
\label{sec:intromethods}
\subsection{Capillary electrophoresis}
The current gold standard technique for STR typing relies on cumbersome capillary electrophoresis methods \cite{ButlerBuelCrivellenteEtAl2004}, which require PCR amplification of the locus of interest, followed by size separation via electrophoresis to determine the alleles present. The method is widely used for typing STRs used as genetic markers for linkage analyses (Marshfield set \cite{BromanMurraySheffieldEtAl1998}), the FBI CODIS set, markers for genealogical studies \cite{ZerjalXueBertorelleEtAl2003,SkoreckiSeligBlazerEtAl1997} and for determining alleles present at known pathogenic loci \cite{LyonLaverYuEtAl2010,BlancoSuarezGandia-PlaEtAl2008}.

Capillary electrophoresis uses a separate reaction per locus, and requires time-consuming optimization of conditions for each reaction. With a cost that is upwards of several dollars per STR and lengthy preparation, current panels can consist of up to several hundreds of STRs - only a fraction of the hundreds of thousands of STR loci in the human genome \cite{Benson1999}. While the method is robust and can achieve accuracy above 99\% \cite{WeberBroman2001}, it has several technical limitations. First, many loci, especially dinucleotide STRs, are plagued by ``stutter peaks'' due to errors introduced during PCR amplification that may complicate calling. Second, this technique can only return the size of the allele: it is unable to distinguish homoplasmic alleles \cite{WeberBroman2001} (two different alleles with the same size but distinct sequences). Third, because capillary techniques simply return the length of the amplified region, it will be sensitive to the presence of linked insertions or deletions that are not part of the STR itself being genotyped. Due to these limitations, electrophoresis is unable to provide an accurate genome-wide picture of sequence variation at STR loci.

\subsection{Genotyping STRs from next-generation sequencing}
\subsubsection{Challenges of genotyping STRs from short reads}
Although theoretically any type of variation should be captured by DNA sequencing, STRs have proven challenging to genotype from short reads produced by high-throughput sequencing platforms. Major challenges include:
\begin{enumerate}
\item Reads must entirely span an STR region to be informative about the number of repeats present.
\item STRs with a large length difference from the reference sequence present as a gapped alignment problem. The run time of mainstream aligners such as BWA \cite{LiDurbin2009a} increases rapidly  with the number of insertions or deletions allows.
\item As in capillary techniques, sequencing technologies require PCR amplification of the DNA sample, which can introduce false ``stutter peaks'' due to the same polymerase slippage process leading to germline STR mutations.
\end{enumerate}
As a result of these challenges, STRs are not routinely analyzed in sequencing studies \cite{TreangenSalzberg2012}, and loci containing repetitive loci are frequently filtered out due to the high presence of genotyping errors in these regions.

\subsubsection{Existing tools for genotyping STRs from sequencing data}
When we began this work, no dedicated tool for genotyping STRs from sequencing data existed. McIver \emph{et al.} \cite{McIverFondonSkinnerEtAl2011} evaluated STR variation in the 1000 Genomes Project \cite{AbecasisAltshulerAutonEtAl2010} samples. However their results were limited to the short variations that could be captured by aligners with poor indel sensitivity, and therefore vastly underestimated polymorphism levels. A major contribution of our work was develop the first efficient algorithm for generating accurate STR genotypes, called lobSTR \cite{GymrekGolanRossetEtAl2012} and described in \autoref{chap:lobstr}. Over the last several years, additional tools have arisen:

\begin{enumerate}
\item \textbf{STRViper} \cite{CaoTaskerWilladsenEtAl2014}: Uses insert size between paired end sequencing reads to detect STR variations.
\item \textbf{RepeatSeq} \cite{HighnamFranckMartinEtAl2013}: uses Bayesian model selection to genotype previously aligned STR-containing reads.
\item \textbf{STR-FM} \cite{FungtammasanAnandaHileEtAl2015}: uses a method based on lobSTR's algorithm but with a modified detection step for increased sensitivity of short repeats and may be applied to non-diploid samples.
\end{enumerate}

The first two tools operate on previously existing alignments, and so are limited by the quality of the upstream aligner. The third may provide improvements to STR calling, especially at homopolymers which are extremely noisy in sequencing data. So far, these tools have not seen widepsread use in mainstream sequence analysis.

In addition to STR-specific callers, a new class of variant callers, including GATK \cite{McKennaHannaBanksEtAl2010} HaplotypeCaller, Platypus \cite{RimmerPhanMathiesonEtAl2014}, and Scalpel \cite{NarzisiOextquotesingleRaweIossifovEtAl2014}, perform local reassembly of diploid haplotypes. These methods can theoretically genotype STRs quite accurately, albeit with greatly increased computational costs. There has so far been no systematic evaluation of their performance at STRs, but these tools may be promising for STR analysis in the future.

\subsubsection{Challenges in visualizing complex variatns}
A key aspect of developing and using tools for variant analysis is visualization of sequence alignments. Often, inspecting raw read alignments can be informative of systamic sequencing artifacts leading to errneous genotype calls.Currently, the UCSC Genome Browser \cite{KentSugnetFureyEtAl2002} and the Integrative Genomics Viewer (IGV) \cite{RobinsonThorvaldsdottirWincklerEtAl2011} are the most widely used genome browsers for alignment visualization.

Visualization of insertions and deletions are key to analyzing reads containing STR variations from the reference genome. However, UCSC, IGV, and similar tools are limited in their ability to display insertions from the reference genome. Because the display is based entirely on the reference genome, insertions are simply displayed as a vertical bar, with no information about the length of insertions. As a result, reads containing different insertions consisting of different lengths, such as a diploid locus where both alleles are longer than the referene allele, will be displayed identically. To overcome this and other genome browser challenges, I have created a novel application, PyBamView, for visualizing sequence alignments at complex variants. This contribution is described in \autoref{chap:pbv}.

\subsection{Long-read technology can capture long repetitive regions}
A major limitation of analyzing STRs from high throughput sequencing data is the short read length. Only reads entirely spanning an STR are informative of the repeat length, and sufficient flanking region on either side of the STR is required for accurate alignment. While the mainstream sequencing technology from Illumina is limited to sequencing at most several hundred base pairs in a single read, alternative sequencing platforms, such as PacBio's SMRT (single molecule real time) sequencing \cite{EidFehrGrayEtAl2009} and the new Nanopore technology \cite{ClarkeWuJayasingheEtAl2009}, can now produce much longer read lengths of up to several thousand base pairs.

These technologies may be used to sequence long repeats observed in expansion disorders such as Fragile X \cite{LoomisEidPelusoEtAl2012} or ataxias \cite{DoiMonjoHoangEtAl2013}, and other complex regions of the genome. Chaisson \emph{et al} \cite{ChaissonHuddlestonDennisEtAl2015} recently applied SMRT to sequence a haploid genome to 40x coverage with an average read length of 5kb. With these long reads, they were able to close 50 gaps in the human genome reference assembly, which were highly enriched for short tandem repeats and other repeats embedded in larger, more complex tandem arrays of degenerate repeats. With longer and longer read lengths, we may soon be able to analyze STRs, as well as other repetitive elements such as variable number tandem repeats (VNTRs) and retrotransposons that were previously inaccessible using sequencing studies.

\section{Population-wide characteristics of STR variation}
\subsection{Previous STR catalogs}
\subsection{Catalogs of other complex variants}

\section{Genetic architecture of gene expression and complex traits}
\label{sec:introarch}
\subsection{Expression quantitative trait loci}
challenges in doing this
GTEx, Pritchard, etc.
\subsection{Additional types of eQTLs}
\subsection{Heritability of gene expression}
Wright, UK10K, Price
\subsection{Power of SNP studies to capture STRs}
Gaurav (here or below)

\section{The role of gene regulation in complex traits}
intro: goal is to understand complex traits
\subsubsection{eQTLs likely mediate complex traits}
overlap with GWAS
\subsubsection{Connecting molecular to disease phenotypes}

\section{Contributions of this thesis}
Taken together, the studies described above provide strong evidence that STRs may play a widespread role in quantitative traits in humans. At the outset of this work, STRs were not amenable to large scale studies due to limitations in bioinformatic and sequencing technologies. Furthermore, repetitive regions were largely considered neutral, or ``junk DNA'' that were simply filtered out of most analyses.

In this thesis, I present our work to enable and perform genome-wide analysis of the effect of STRs on gene expression, and ultimately complex traits, in humans. These efforts have shown that STRs indeed contribute significantly to the heritability of gene expression and likely have a widespread effect on genome regulation. Importantly, the tools and results generated by this work have helped to renew interest in variants beyond point mutations and have highlighted the importance of including repetitive variations in genome-wide studies of complex traits. 

Specifically, I present the following studies, briefly summarized here:

\subsection{Tools for STR analysis from short reads}
STRs pose major challenges to current bioinformatics pipelines. They are often extremely polymorphic and exhibit large length differences from the reference sequence. Additionally, they are prone to stutter errors that occur due to polymerase slippage during PCR amplification. As the first step to studying STRs on a large scale, we developed lobSTR \cite{GymrekGolanRossetEtAl2012} \autoref{chap:lobstr}, an algorithm for profiling short tandem repeats from high throughput sequencing data. lobSTR employs a unique mapping strategy to rapidly align repetitive STR containing reads, and uses statistical learning techniques to account for stutter noise.

We have invested significant effort in making this tool user friendly, and lobSTR now has an active online user community on Github and Google groups. lobSTR has been used to generate large scale catalogs of STR variation from the 1000 Genomes and other projects (see below). We continue to improve lobSTR to adapt to improvements in sequencing technology and bioinformatics tools. For instance, lobSTR now handles input from the BWA-MEM aligner, has a specific noise model for PCR-free protocols, and uses joint calling to simultaneously genotype thousands of samples at once.

In addition to lobSTR itself, we are actively developing additional tools for analysis and visualization of STR and other complex variant data. To deal with challenges in visualization STRs and other complex variants, I developed PyBamView \cite{Gymrek2014} \autoref{chap:pbv}, an interactive alignment visualization tool. PyBamView allows accurate visualization of insertions, which are not displayed well by current browsers such as IGV and UCSC, and allows researchers to share alignment views with others through a web browser.

\subsection{The first genome-wide catalogs of STR variation}
Until recently, studies of STR variation have been limited to several highly ascertained sets of loci encompassing several thousand highly polymorphic autosomal and Y-chromosome STRs used in linkage studies, genetic genealogy, and forensics. Little is known about patterns of polymorphism across the majority of the 1.6 million STR loci in the human genome. Building a catalog of STR variation is an important first step in determining their contribution to genetic and phenotypic variation.

To build an initial genome-wide STR catalog, we applied lobSTR to more than 3,000 low coverage whole genome sequencing datasets generated by the 1000 Genomes Project \cite{WillemsGymrekHighnamEtAl2014} \autoref{chap:catalog}. We used this call set to analyze sequence determinants of STR variation, assess patterns of variation in coding regions, and find common loss of function alleles. We found that hundreds of thousands of STRs are polymorphic in the number of repeats across individuals. This catalog has already served as a valuable resource for researchers interested in variation at specific STR loci that may be implicated in human phenotype. 

While our initial catalog gave accurate information on the distribution of STR alleles across populations, the quality of individual genotypes was poor due to very low sequencing coverage. Additionally, we were not able to obtain accurate genotypes for homopolymers, which show notoriously high error rates due to polymerase slippage. We recently created the most comprehensive STR call set to date based on 300 deeply sequenced genomes from the Simons Genome Diversity Project \autoref{chap:sgdp}. These calls show 93\% concordance with gold standard STR genotypes from capillary electrophoresis, and accurately capture population structure. This dataset provides unprecedented opportunities to study STR variation that were not possible using previous studies either due to the small number of markers or to the low quality of individual genotypes, including in-depth study of homopolymers and population dynamics of STR variation.

\subsection{Abundant contribution of STRs to gene expression in humans}
With a robust pipeline for STR genotyping and a large catalog of STR genotypes, we were in a position to assess the genome-wide contribution of STRs to phenotypes for the first time. We focused on the role of STRs in regulating gene expression. Multiple single-gene studies in humans and other organisms have shown that STRs may modulate gene expression in \emph{cis}. However there has been no systematic study of their effect on expression in humans.

We conducted a genome-wide analysis of STRs that affect expression of nearby genes, which we termed expression STRs (eSTRs), in lymphoblastoid cell lines (LCLs) \cite{GymrekWillemsGuilmatreEtAl2015} \autoref{chap:estr}. This well-studied model permitted the integration of whole genome sequencing data, expression profiles from RNA-sequencing and arrays, and functional genomics data. We tested for association in close to 190,000 STR$\times$gene pairs and found over 2,000 significant eSTRs. Using a multitude of statistical genetic and functional genomics analyses, we show that hundreds of these eSTRs are predicted to be functional, and that 10-15\% of \emph{cis} heritability of gene expression in LCLs can be attributed to STRs, uncovering a new class of regulatory variants.

\subsection{Identifying personal genomes by surname inference}
Sequencing datasets are often shared without identifiers under the assumption that the identities of the samples will not be revealed. We showed that we can recover genealogical STRs from the Y chromosome by applying lobSTR to whole genome sequencing datasets. We used the resulting genotypes to query recreational genealogical databases to recover the surnames of anonymous sample donors. Combining the surname with additional metadata such as age and state can lead to a complete breach of anonymity \cite{GymrekMcGuireGolanEtAl2013} \autoref{chap:surname}.

Due to the widespread use of STRs in forensics and genealogy, this work had important implications for genetic privacy of human research subjects. As a result I have become more aware of ethical concerns and have been an active participant in discussions of social aspects of genetic research in the wider genomics community. Importantly, it is now recognized that it is impossible to promise anonymity, and that this should be clarified to participants. As a result of our study many data usage agreements have been augmented to prevent researchers from attempting to identify participants. We believe this project paved the way toward greater transparency about data privacy issues and will ultimately help protect and inform participants in genetic research.
